sql/moz-fx-data-shared-prod/firefox_accounts/fxa_users_services_daily_summary/view.sql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_accounts.fxa_users_services_daily_summary`
+AS
+SELECT
+  * REPLACE (cn.code AS country),
+  fxa.country AS country_name
+FROM
+  `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_users_services_daily_summary_v1` AS fxa
+LEFT JOIN
+  `moz-fx-data-shared-prod.static.country_names_v1` AS cn
```

Comment:
Doing this join in the Looker Model enables having also region and subregion, in case is important now or for a future request.

Reply:
I'm just following the same structure/pattern used for `sql/moz-fx-data-shared-prod/firefox_accounts/fxa_users_services_daily/view.sql`